integration-cli/docker_cli_version_test.go

```diff
@@ -26,3 +26,28 @@ func (s *DockerSuite) TestVersionEnsureSucceeds(c *check.C) {
 		}
 	}
 }
+
+// ensure the Windows daemon return the correct platform string
+func (s *DockerSuite) TestVersionPlatform_w(c *check.C) {
+	testRequires(c, DaemonIsWindows)
+	testVersionPlatform(c, "windows/amd64")
+}
+
+// ensure the Linux daemon return the correct platform string
+func (s *DockerSuite) TestVersionPlatform_l(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	testVersionPlatform(c, "linux/amd64")
+}
+
+func testVersionPlatform(c *check.C, platform string) {
+	out, _ := dockerCmd(c, "version")
+
+	// Verify the second 'OS/Arch' matches the platform
+	split := strings.Split(out, "\n")
+	if len(split) != 16 {
```

Comment:
this test is failing in experimental. We should really not count lines in an output and check that the content that we want is there.